packages/eslint-plugin-pf-codemods/src/rules/v6/emptyStateHeaderMoveIntoEmptyState/emptyStateHeader-move-into-emptyState.ts

```diff
@@ -204,6 +174,10 @@ module.exports = {
           "EmptyStateIcon"
         );
 
+        if (!header && !titleChild && !emptyStateIconChild) {
+          return;
+        }
+
```

Comment:
This addition has to do with adding this valid test: `import { EmptyState } from "@patternfly/react-core"; <EmptyState>Foo bar</EmptyState>`, which was previously in the invalid array. I think we don't have to throw an error in such case (when consumers are using empty state without these props, so there is nothing to fix - well there would be a fix we don't want, adding an empty text, because `newEmptyStateProps` is empty), so that's why the early return. But it is a very rare case anyway.